### PR TITLE
[KHRGA-123] Updated additions of handler

### PR DESCRIPTION
**Pull request by imvucic**
Issue: KHRGA-123

### CODE REVIEW

File: source/iface/host-task-add-to-handler.rst [hunk 1]
```diff
@@ -5,3 +5,31 @@
...
+to invoke ``hostTaskCallable`` exactly once. The scheduling of the
+invocation of ``hostTaskCallable`` in relation to other commands
+enqueued to the SYCL runtime must be in accordance with the dependency
+model described in |SYCL_SPEC_MEM_MODEL|. Initializes an
+``interop_handle`` object and passes it to ``hostTaskCallable``
+when it is invoked if ``std::is_invocable_v<T, interop_handle>``
+evaluates to true, otherwise invokes ``hostTaskCallable``
+as a nullary function.
+
+.. seealso:: :ref:`handler`
```
Comment (mkinsner) [inline source/iface/host-task-add-to-handler.rst:35]: Should this link to the SYCL spec?  Currently links within the SYCL reference.

File: source/iface/host-task-add-to-handler.rst [hunk 2]
```diff
@@ -5,3 +5,31 @@
...
+
+This section describes member functions in the command group handler
+class that are used with host tasks.
+
+::
+
+  class handler {
+    ...
+
+        public
```
Comment (mkinsner) [inline source/iface/host-task-add-to-handler.rst:17]:
```suggestion
        public:
```

File: source/iface/host-task-add-to-handler.rst [hunk 3]
```diff
@@ -5,3 +5,31 @@
...
+        : template <typename T>
```
Comment (mkinsner) [inline source/iface/host-task-add-to-handler.rst:18]:
```suggestion
        template <typename T>
```

File: source/iface/host-task-add-to-handler.rst (hunk 2 above)
Comment (mkinsner) [reply source/iface/host-task-add-to-handler.rst:17]: https://github.com/KhronosGroup/SYCL-Docs/pull/527

File: source/iface/host-task-add-to-handler.rst (hunk 1 above)
Comment (imvucic) [reply source/iface/host-task-add-to-handler.rst]: Added only link to SYCL spec with https://github.com/KhronosGroup/SYCL_Reference/pull/153/commits/8322c7638e0e30a13a089dd315d66ee179e33738

File: source/iface/host-task-add-to-handler.rst (hunk 2 above)
Comment (imvucic) [reply source/iface/host-task-add-to-handler.rst]: Fixed with https://github.com/KhronosGroup/SYCL_Reference/pull/153/commits/8322c7638e0e30a13a089dd315d66ee179e33738

File: source/iface/host-task-add-to-handler.rst (hunk 3 above)
Comment (imvucic) [reply source/iface/host-task-add-to-handler.rst]: Fixed with https://github.com/KhronosGroup/SYCL_Reference/pull/153/commits/8322c7638e0e30a13a089dd315d66ee179e33738

File: source/iface/host-task-add-to-handler.rst [hunk 4]
```diff
@@ -5,3 +5,31 @@
...
+          void
+          host_task(T&& hostTaskCallable); // (1)
```
[the remark this answers is not in the document]
Comment (imvucic) [reply source/iface/host-task-add-to-handler.rst]: Fixed with https://github.com/KhronosGroup/SYCL_Reference/pull/153/commits/8322c7638e0e30a13a089dd315d66ee179e33738

**Review by mkinsner — APPROVED** [review]

**Pull request merged**